### PR TITLE
Remove perl dependency

### PR DESCRIPTION
**Pull request by CyBeRoni**
At some point someone thought it was a good idea to fix non-linux compatibility by using perl to do an in-place find-and-replace. Unfortunately that of course brings with it a dependency on perl which is not always available in minimal environments.

This patch re-replaces perl with sed and I believe does so in a POSIX-compatible way.

### CODE REVIEW

File: src/scripts/helpers/zotonic_setup [hunk 1]
```diff
@@ -40,8 +40,7 @@ export -f require_zotonic_not_running
 function enablesite {
     config="priv/sites/$1/config"
     expr="s/enabled, \(true\|false\)/enabled, $2/"
-    perl -e "$expr" -pi $config
-    #echo sed -e "$expr" -i $config
+    sed -e "$expr" -i.bak -e $config
```
Comment (ddeboer) [inline src/scripts/helpers/zotonic_setup]: Is `sed -i` portable?

Comment (arjan) [inline src/scripts/helpers/zotonic_setup]: I see that on [BSD](https://www.freebsd.org/cgi/man.cgi?query=sed) it's supported.

Comment (arjan) [inline src/scripts/helpers/zotonic_setup]: But shouldn't this line read: `sed -e "$expr" -i $config` ?

Comment (CyBeRoni) [inline src/scripts/helpers/zotonic_setup]: sed -i with an extension works on the platforms I've tried (sunos, osx, linux) but it don't see the 'enablesite' function actually being called anywhere, plus it hardcodes the sites dir as being under priv/ so I didn't give it as much thought as the main config file. I actually considered removing it entirely.

Comment (arjan) [inline src/scripts/helpers/zotonic_setup]: Agreed, it might be a leftover from long ago, you have my blessing to remove it :)

**Comment by CyBeRoni** [comment]
Ok I removed that old enablesite function.

**Comment by ddeboer** [comment]
Thanks, @CyBeRoni!

**Pull request merged**